tests/python/unittest/test_meta_schedule_tune_relay.py

```diff
@@ -149,6 +149,7 @@ def test_meta_schedule_tune_relay(
             config=ReplayTraceConfig(
                 num_trials_per_iter=32,
                 max_trials_per_task=32,
+                max_trials_global=32,
```

Comment:
@zxybazh See https://github.com/apache/tvm/pull/10366/files#r840995932. I fixed this use of `ReplayTraceConfig`, but is this the right value?